service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -143,6 +147,15 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
         sendNotification(STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE, email, parameters, reference);
     }
 
+    @EventListener
+    public void sendNotificationsForIssuedCaseManagementOrder(CMOEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = (String) caseDetails.getData().get(CASE_LOCAL_AUTHORITY_PROPERTY_NAME);
+
+        sendCMONotificationForLocalAuthority(caseDetails, localAuthorityCode);
+        // TODO: 10/12/2019 FPLA-27 will add document related notifications
```

Comment:
Is this comment necessary?

Reply:
yes. It was a reminder to implement as part of FPLA-27, as it will be branched off this branch. Will be removed in FPLA-27 PR.